### PR TITLE
parse enums 

### PR DESCRIPTION
**Pull request by saiedkia**

### CODE REVIEW

File: src/UrlFilter/UrlFilter.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.3</TargetFramework>
```
Comment (jesseh77) [inline src/UrlFilter/UrlFilter.csproj:4]: I don't see anything here that requires bumping the netstandard version

File: tests/UrlFilter.Tests/TestDocument.cs [hunk 2]
```diff
@@ -27,5 +28,13 @@ public TestDocument(int seed, bool includeSubdocument = false)
         public string MoreText { get; set; }
         public TestDocument SubDocument { get; set; }
         public List<TestDocument> DocumentCollection { get; set; }
+        public State State { get; set; }
+    }
+
+    enum State
```
Comment (jesseh77) [inline tests/UrlFilter.Tests/TestDocument.cs:34]: Name it something inline with the other properties please.

**Review by jesseh77 — COMMENTED** [review]
Thanks.  Add a test for the ValueProcessor as well please and don't bump the netstandard version.

File: src/UrlFilter/UrlFilter.csproj (hunk 1 above)
Comment (saiedkia) [reply src/UrlFilter/UrlFilter.csproj:4]: [IsEnum](https://github.com/jesseh77/UrlFilter/pull/10/commits/d3035f7135dea7bfe704f62c50107abaa934c187#diff-2ce623559bd4c2710dbb0b2747f7c411R92-R100#L93) property is available in netstandard 2.1 and higher

**Pull request closed without merging**